src/frontend/next/src/pages/search.tsx

```diff
@@ -0,0 +1,13 @@
+import SEO from '../components/SEO';
+import SearchPage from '../components/SearchPage';
+
+const Search = () => {
+  return (
+    <>
+      <SEO pageTitle="Search" />
+      <SearchPage />
+    </>
+  );
```

Comment:
Can you add a mock `<SearchPage/>` so we can pass the CI/CD test?

Reply:
I'm going to rebase this once SearchPage lands, I think adding in a mock component will just make that component unnecessarily harder to land